docs/users/SystemRequirements.md

```diff
@@ -1,23 +1,52 @@
 # System Requirements
-!!! note "This page does not include the requirements to run any supported games. Please refer to the game store page for details."
+!!! info "This page does not include the requirements to run any supported games. Please refer to the game store page for details."
 
-This page lists the recommended system requirements to install and run the app on all supported platforms. More details will be added later as the app is tested on different systems and configurations.
+This page lists the recommended system requirements to install and run the app on all supported platforms.
 
 ## Windows 
 
-* Windows 10 or 11
+!!! example "Testing required"
+    We are currently testing the app on a variety of different systems and will provide more detailed system requirements in future.
 
-## Linux
+|| Recommended |
+| --- | --- |
+| OS | Windows 10/11 with updates (64-bit OS required) |
+<!-- | Processor | A multi-core CPU is recommended |
+| Memory | More than 4 GB RAM |
+| Storage | 5GB available space* |
+
+<sub>\* Storage space required does not factor in the number of games managed, the size of the managed games and the mods installed.</sub> -->
+
+## SteamOS + Linux 
+!!! example "Testing required"
+    We are currently testing the app on a variety of different systems and will provide more detailed system requirements in future.
+
+|| Recommended |
+| --- | --- |
+| OS | A currently supported Linux distribution |
+<!-- | Processor | A multi-core CPU is recommended |
+| Memory | More than 4 GB RAM |
+| Storage | 5GB available space* |
+
+<sub>\* Storage space required does not factor in the number of games managed, the size of the managed games and the mods installed.</sub> -->
+
+### Linux Dependencies
 The App requires the following programs to be available at runtime:
 
 - `xdg-open` to open URLs in your browser.
 - `update-desktop-database` to update the MIME cache after creating a desktop entry file for the NXM link handler.
 
 If you plan to install using the AppImage, make sure you have the following dependencies installed:
 
+- [FUSE 2](https://github.com/AppImage/AppImageKit/wiki/FUSE) is required to run any AppImage.
 - `glibc` 2.2.5 or newer
 - `glibc++` 3.4 or newer
```

Comment:
`glibc` and `glibc++` are always required. FUSE 2 is required for the AppImage.

Reply:
This is your text, I just copy/pasted it.

Reply:
It's the text above that's an issue

> If you plan to install using the AppImage, make sure you have the following dependencies installed:

Can probably just be something like this:

```markdown
The App requires the following dependencies:

- `glibc` 2.2.5 or newer
- `glibc++` 3.4 or newer
- [FUSE 2](https://github.com/AppImage/AppImageKit/wiki/FUSE) if you plan on using the AppImage.
```